peps/pep-0778.rst

```diff
@@ -0,0 +1,352 @@
+PEP: 778
```

Comment:
778 is already assigned for "Supporting Symlinks in Wheels": https://github.com/python/peps/pull/3786

768 is the next available in sequence:
```console
❯ pip install pepotron --quiet && pep next
Next available PEP: 768
```

```suggestion
PEP: 768
```